src/main/routes/home.ts

```diff
@@ -2,8 +2,9 @@ import { Application } from 'express';
 
 export default function(app: Application): void {
 
-  app.get('/', (req, res) => {
-    res.render('home');
+  app.get('/', (req: any, res: any) => {
+    const data = req.i18n.getDataByLanguage(req.lng).home;
```

Comment:
I wonder if we should render everything at boot then access it during the request. It's less error prone and faster.

For probate I did something like

```
const pagesByLanguage = {
  'en': render('page', 'en'),
  'cy': render('page', 'cy'),
}

app.get('/', (req: any, res: any) => {
    const data = pagesByLanguage(req.lng);
    ...
});
```

That won't work here because the i18n lib is attached to the req. We should be able to work around that though.... I hope.

Maybe it is also attached to `app` in the function call

Reply:
I guess this wouldn't be possible in most cases as the template would be dynamic and need data from the API or some such place

Reply:
I guess we can review this going forward.